modules/README.md

```diff
@@ -146,4 +149,53 @@ interface ISafeProtocol712SignatureValidator {
 }
 ```
 
+- Signature Validator Manager
+
+
+```solidity
+interface ISafeProtocolSignatureValidatorManager {
+    /**
+     * @param dataHash Hash of the data that is signed
+     * @param data Arbitrary data containing the following layout:
+     *             Layout of the data:
+     *              0x00 - 0x04: selector
```

Comment:
Do we have any more information on this layout? 

From a first glance I would also argue that it is missing the signature

Reply:
updated interface